src/lib/tusktask/optimisticUpdates/registerNewMessageToConversation.ts

```
@@ -0,0 +1,43 @@
+) => {
+  if (!selectedRoom || !content) return;
+
+  const oldData = queryClient.getQueryData([
+    "conversation",
+    selectedRoom,
+  ]) as DetailConversationGetResponse;
+
+  queryClient.setQueryData(["conversation", selectedRoom], () => {
+    if (!oldData?.data) return oldData;
+
+    let messages = [...oldData?.data?.messages];
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)